src/xo-app/dashboard/health/index.js

```diff
@@ -240,42 +250,50 @@ const ALARM_COLUMNS = [
 export default class Health extends Component {
   componentWillReceiveProps (props) {
     if (props.alertMessages !== this.props.alertMessages) {
-      Promise.all(
-        map(props.alertMessages, ({ body }) => {
-          const matches = /^value: ([0-9.]+) config: (.*)$/.exec(body)
-          if (!matches) {
-            return
-          }
+      this._updateAlarms(props)
+    }
+  }
+
+  componentDidMount () {
+    this._updateAlarms(this.props)
+  }
+
+  _updateAlarms = props => {
+    Promise.all(
+      map(props.alertMessages, ({ body }, id) => {
+        const matches = /^value: ([0-9.]+) config: (.*)$/.exec(body.replace(/\n/g, ' '))
```

Comment:
`.` does not handle new lines, if you want to match everything, you can do: `[^]`.